packages/graphql-language-service-server/src/GraphQLLanguageService.ts

```diff
@@ -96,11 +100,38 @@ export class GraphQLLanguageService {
     this._logger = logger;
   }
 
-  getConfigForURI(uri: Uri) {
-    const config = this._graphQLCache.getProjectForFile(uri);
-    if (config) {
-      return config;
+  getAllProjectsForFile(uri: Uri) {
+    const filePath = uri.startsWith('file:') ? fileURLToPath(uri) : uri;
```

Comment:
Passing an Uri that starts with the `file://` protocol doesn't work properly in `grahiQLConfig.getProjectForFile`. Internally that calls `minimatch` and it always returns false.

Reply:
yes there was a breaking change in vscode some years ago, and there are a few places that we didn't handle this, thanks for the bugfix! I will come back around look at the implementations because they should all be passing a file url